unit_tests/test_data_utils.py

```diff
@@ -0,0 +1,11 @@
+def test_remove_html_tags():
+    '''Unit test of data utility'''
+    from jag.utils.data_utils import remove_html_tags
```

Comment:
any reason to have the import here instead of at the top of the file?

Reply:
I moved the import at the top of the files.